server/routes/coordinator.js

```diff
@@ -57,6 +57,9 @@ module.exports = function Index({ reportService, involvedStaffService, reviewSer
 
     confirmDeleteReport: async (req, res) => {
       const { reportId } = req.params
+      const referringPage = req.header('Referer') || 'confirm-delete'
```

Comment:
I would generally avoid using referrer if you can get away with it as its only transient information - any refresh or redirection will remove it

Reply:
changed

Reply:
done